packages/asl-pages/pages/project/read/views/index.jsx

```diff
@@ -135,7 +138,8 @@ export default function ProjectLandingPage() {
         title={<Snippet>{`page.title.${project.status === 'inactive' ? 'application' : 'granted'}`}</Snippet>}
         subtitle={project.title || 'Untitled project'}
       />
-
+      {flashMessage && (<FlashBanner className="flash-banner--success" heading="HBA file replaced"
+        body={flashMessage} />)}
```

Comment:
I don't think we want to be assuming where the flash message has come from and what it is. The flash banner itself should be reading the content with it's own useSelector.
```suggestion
      <FlashBanner />
```

Rename the current FlashBanner to AlertBanner to match the design system name.

Then FlashBanner encapsulates the behaviour of using the alert banner to display a flash message:

```js
export function FlashBanner() {
  const flash = useSelector(state => state.static.flash);
  if(!flash) {
    return null;
  }

  const {title, body, type} = flash;
  
  return <AlertBanner className={`flash-banner--${type}`} heading={title} body={body} />;
}
```

Reply:
looking at useSelector in asl-component...

Reply:
refactored @jeff-horton-ho-sas .